client/src/components/Visualizations/PluginList.vue

```diff
@@ -137,7 +136,8 @@ export default {
             if (plugin.target == "_top") {
                 window.location.href = href;
             } else {
-                $("#galaxy_main").attr("src", href);
+                var galaxy_main_el = document.getElementById("galaxy_main");
```

Comment:
Thanks for working on this!  One minor nit that linting will complain about:

```suggestion
                const galaxy_main_el = document.getElementById("galaxy_main");
```